stock_markets.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+class StockMarket():
```

Comment:
I would recommend that in future versions we could have a StockMarket base class and separate child classes for each supported stock markets